brd-ios/breadwallet/src/ViewControllers/KYC/KYCForgotPassword/KYCForgotPasswordInteractor.swift

```diff
@@ -51,7 +51,9 @@ class KYCForgotPasswordInteractor: KYCForgotPasswordBusinessLogic, KYCForgotPass
         var validationValues = [Bool]()
         validationValues.append(!email.isNilOrEmpty)
         validationValues.append(Validator.validateEmail(value: email ?? "", completion: { [weak self] isViable in
-            self?.presenter?.presentValidateField(response: .init(isViable: isViable, type: .email))
+            guard let email = self?.email else { return }
+            
+            self?.presenter?.presentValidateField(response: .init(isViable: isViable, type: .email, isFieldEmpty: email.isEmpty))
```

Comment:
you can just to

var isViabled = isViable
isViable = email.isEmpty ? false : isViable

and get rid of isFieldEmpty: email.isEmpty completely.